src/level/parser/parserparam.cpp

```diff
@@ -339,6 +339,10 @@ ObjectType CLevelParserParam::ToObjectType(std::string value)
     if (value == "Portico"           ) return OBJECT_PORTICO;
     if (value == "SpaceShip"         ) return OBJECT_BASE;
     if (value == "PracticeBot"       ) return OBJECT_MOBILEwt;
```

Comment:
This should seriously change to be the 'alias', but it seems impossible without breaking levels. How about at least a warning for the sake of 'unification'?

Reply:
Keeping this in Scene is intentional, you can use both PracticeBot and WheeledTrainer for the sake of compatibility

Reply:
I know, I'm just suggesting to deprecate one of them with a warning because it's confuzing

Reply:
I don't think deprecation is needed and I don't really see it as confusing. Besides, a warning can always be added later and since the pull request is already too old, I'm resolving this conversation.